### DISCUSSION TITLE
Error: The remote server returned an error: (502) Bad Gateway

### DISCUSSION THREAD

**Discussion by kdham** [builtbybel/FlyOOBE#37] [General]
I start the process using Flyby11 and when it trys to download the ISO I get the error noted in the subject line. " Error: The remote server returned an error: (502) Bad Gateway"
Any advice.
Thank you.

**Comment by Belim** [comment]
https://github.com/builtbybel/Flyby11/releases/tag/0.17.160

**Reply by kdham** [reply]
I'm away for a few days but when I get back in a day or so I will try it. And get back with my results.
Regards and 
Thank you

**Reply by kdham** [reply]
Thank you. 
I'm back and yes that worked Thank you Very Much.
I have a couple of systems Im now going to upgrade
Happy Christmas.

**Reply by Belim** [reply]
Merry Christmas to you too!
